velox/exec/Task.h

```diff
@@ -485,6 +485,15 @@ class Task : public std::enable_shared_from_this<Task> {
     return mutex_;
   }
 
+  /// Returns the number of created and deleted tasks since the velox engine
+  /// starts running so far.
+  static uint64_t numCreatedTasks() {
+    return numCreatedTasks_;
+  }
+  static uint64_t numDeletedTasks() {
```

Comment:
nit: Add empty line and split the comment so each method has its own documentation.